src/core/tsi/ssl_transport_security.cc

```diff
@@ -1507,6 +1530,26 @@ tsi_result tsi_create_ssl_server_handshaker_factory_ex(
                                     &pem_key_cert_pairs[i], cipher_suites);
       if (result != TSI_OK) break;
 
+      // Allow client cache sessions.
```

Comment:
By doing so, all SSL servers will allow session resumption. Maybe we shall provide a way for customer who want to opt out session resumption.

Reply:
Tickets are enabled on server side by default in boringssl and broken in openssl (this line fixes it for openssl only and is no-op for boringssl). What do you think if I leave a TODO here to add ability to disable/control/rotate session tickets & we will solve it as part of session ticket support on server side (issue 14431)?

Reply:
Sounds good to me. Please add a TODO.

Reply:
I've updated a cl with TODO, also elaborated a bit more about openssl in the comment.